prql-js/src/lib.rs

```diff
@@ -2,7 +2,7 @@
 #![allow(clippy::drop_non_drop)]
 mod utils;
 
-use prql_compiler::{pl_to_json, pl_to_prql, translate, ErrorMessage, IntoErrorMessage};
+use prql_compiler::{pl_to_json, pl_to_prql, compile, ErrorMessage, IntoErrorMessage};
```

Comment:
How should we deal with `options` on the language bindings? Would we need an `Options.default()` for each language?

Would there be any logic in having `Option<Options>`, and then we could easily pass a `None` value? It possibly adds a layer of indirection within rust though.

Reply:
Good idea, will do that.